.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
+   "tabWidth": 2,
+   "singleQuote": false
```

Comment:
Uwazi does single quotes, so I'd prefer to do the same here

Reply:
done, but not getting it to work for json files.